src/Sentry/Android/JavaExtensions.cs

```diff
@@ -0,0 +1,163 @@
+using System;
+using System.Linq;
+using System.IO;
+using System.Text.Json;
+using Sentry.Extensibility;
+using JavaInputStreamReader = Java.IO.InputStreamReader;
+using JavaOutputStreamWriter = Java.IO.OutputStreamWriter;
+
+namespace Sentry.Android
+{
+    internal static class JavaExtensions
+    {
+        public static SentryLevel ToSentryLevel(this Java.SentryLevel level)
+        {
+            // note: switch doesn't work here because JNI enums are not constants
+            if (level == Java.SentryLevel.Debug)
+                return SentryLevel.Debug;
+            if (level == Java.SentryLevel.Info)
+                return SentryLevel.Info;
+            if (level == Java.SentryLevel.Warning)
+                return SentryLevel.Warning;
+            if (level == Java.SentryLevel.Error)
+                return SentryLevel.Error;
+            if (level == Java.SentryLevel.Fatal)
+                return SentryLevel.Fatal;
+
+            throw new ArgumentException(null, nameof(level));
+        }
+
+        public static Java.SentryLevel ToJavaSentryLevel(this SentryLevel level) =>
+            level switch
+            {
+                SentryLevel.Debug => Java.SentryLevel.Debug!,
+                SentryLevel.Info => Java.SentryLevel.Info!,
+                SentryLevel.Warning => Java.SentryLevel.Warning!,
+                SentryLevel.Error => Java.SentryLevel.Error!,
+                SentryLevel.Fatal => Java.SentryLevel.Fatal!,
+                _ => throw new ArgumentException(null, nameof(level))
```

Comment:
Good case for ArgumentOutOfRange?
Curious why `null` instead of level as an arg. Could we use it? Would be useful to know what level was given.